data/api/server-server/query.yaml

```diff
@@ -121,7 +121,7 @@ paths:
       parameters:
         - in: query
           name: user_id
-          description: The user ID to query.
+          description: The user ID to query. Must be a user local to the receiving homeserver.
```

Comment:
this is already mentioned in the description. I suppose there is no harm in repeating it.